src/index.html

```diff
@@ -11,6 +11,47 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <script type="text/javascript" src="./main.js"></script>
+    <form
+      action="#"
+      method="post"
+      data-qa="big"
+    >
+      <div class="row search">
```

Comment:
```suggestion
      <div class="search-wrapper">
```
and add all styles you need to `search-wrapper` class in the CSS (instead of having 2 different classes)

moreover, you may get rid of this wrapper and add some class to `form` and apply styles you need to it